torchio/transforms/augmentation/intensity/random_simulation.py

```diff
@@ -0,0 +1,172 @@
+
+from typing import Union, Tuple, Optional, Dict, List
+import torch
+from ....torchio import DATA, TypeData, TypeRangeFloat, TypeNumber, AFFINE, INTENSITY, TYPE, LABEL
+from ....data.subject import Subject
+from ....data.image import Image
+from .. import RandomTransform
+
+MEAN_RANGE = (0.1, 0.9)
+STD_RANGE = (0.01, 0.1)
+
+
+class RandomSimulation(RandomTransform):
+    r"""Generate a random image from label maps.
+
+    Args:
+        label_keys: Keys designating the label images in the sample that
+            will be used to generate the new image.
+            If a single string is given, the corresponding label map is expected
+            to be a label encoding map.
+            Default will use all images labeled as label in the sample.
+        image_key: Key to which the new volume will be saved.
+            If this key corresponds to an already existing volume, zero elements from
+            the new volume will be filled with elements of the original volume.
+        coefficients: Dictionary containing the mean and standard deviation for
```

Comment:
Maybe "coefficients" is not the best name, as it sounds like factors of a polynomial. Maybe "gaussian_parameters"?